internal/commands/download_sbom.go

```diff
@@ -0,0 +1,47 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	cpkg "github.com/buildpacks/pack/pkg/client"
+	"github.com/buildpacks/pack/pkg/logging"
+)
+
+type DownloadSBOMFlags struct {
+	Local          bool
+	Remote         bool
+	DestinationDir string
+}
+
+func DownloadSBOM(
+	logger logging.Logger,
+	client PackClient,
+) *cobra.Command {
+	var flags DownloadSBOMFlags
+	cmd := &cobra.Command{
+		Use:     "download-sbom <image-name>",
```

Comment:
Can I throw in my nitpickiness here?

Can we find a better way to scope this to subcommands as opposed to having a hyphenated command? Do we envision other sbom operations? Even if we don't, what about `pack sbom download` instead?

Reply:
The only other subcommand I can think of is `pack sbom attach` pending https://github.com/buildpacks/rfcs/pull/186. I don't feel strongly here though, I can change it.

Reply:
_Addressed in latest commit._